pkg/api/v1beta4/dynakube/oneagent/props.go

```diff
@@ -103,8 +104,8 @@ func (oa *OneAgent) GetConnectionInfoConfigMapName() string {
 	return oa.name + OneAgentConnectionInfoConfigMapSuffix
 }
 
-func (oa *OneAgent) IsReadOnlyOneAgentsMode() bool {
-	return oa.IsCloudNativeFullstackMode() || (oa.IsHostMonitoringMode() && oa.IsCSIAvailable())
+func (oa *OneAgent) IsOneAgentModeSupportingReadOnlyFS() bool {
```

Comment:
```suggestion
func (oa *OneAgent) IsReadOnlyFSSupported() bool {
```
you are in the `oneagent` package, writing a method for the `OneAgent` struct, I don't think we need more stuttering 😅